edweiss-app/model/lectures/lectureDoc.ts

```diff
@@ -10,37 +10,36 @@
 
 import { FunctionFolder, FunctionOf } from '../functions';
 import Quizzes from '../quizzes';
-import { Timestamp } from '../time';
 
 // ------------------------------------------------------------
 // -------------------   Lecture Namespace   ------------------
 // ------------------------------------------------------------
 
 namespace LectureDisplay {
 
-    export type LectureEvents = "quiz";
-
-    interface LectureEventBase {
-        type: LectureEvents;
-        done: boolean;
-        pageNumber: number;
-    }
-
-    export interface QuizLectureEvent extends LectureEventBase {
-        type: "quiz";
-        quizModel: Quizzes.Quiz;
-    }
-
-    export interface Lecture {
-        pdfUri: string;
-        nbOfPages: number;
-        availableToStudents: boolean;
-        audioTranscript: { [pageNumber: number]: string; };
-    }
-
-    export const Functions = FunctionFolder("lectures", {
-        addAudioTranscript: FunctionOf<{ courseId: string, lectureId: string, pageNumber: number, transcription: string; }, {}, 'invalid_arg' | 'error_firebase' | 'successfully_added'>("addAudioTranscript"),
-    });
+	export type LectureEvents = "quiz";
```

Comment:
What was the change here? Some code is highlighted in green, even though I do not see any changes with the last version (in red).

Reply:
It's probably tabulations vs spaces.